nextflow.config

```diff
@@ -97,6 +97,7 @@ params {
     ichorcna_reptime_wig                = null
     // FIXME: Supply it with the pipeline
     ichorcna_centromere_file            = null
+    correct_ichorcna_plot               = false
```

Comment:
I'm not a fan of this option name, it implies ichorCNA does things "wrong". I know it means something else, but it's ambiguous  Also it's not in the style of the other options (`tool_name_OPTION`). 

I would suggest `ichorcna_ploidy_aware_plot` or somesuch.